### PR TITLE
Add test-coverage report

### PR DESCRIPTION
**Pull request by SimenB**
I'm on a roll today! :laughing: 

This prints out a test-coverage report. I added the stuff for it to work on Coveralls as well. If you don't want that, I can make a PR with just the coverage stuff (if that is interesting).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,4 @@ node_js:
   - iojs
   - 0.12
   - 0.10
+#after_success: cat coverage/lcov.info | node_modules/.bin/coveralls --verbose
```
Comment (SimenB) [inline .travis.yml]: After you log in to https://coveralls.io, removing the pound sign should be enough to generate coverage reports

**Comment by lazd** [comment]
Ha, you're awesome! This is probably cleaner than how I have coverage setup on other repositories too.

**Comment by SimenB** [comment]
@lazd ping

**Comment by lazd** [comment]
It's workin'! https://coveralls.io/r/lazd/gulp-csslint

**Comment by SimenB** [comment]
Great! I'll whip up a PR with the badge :smile:

**Pull request merged**